pkg/controller/operators/olm/operator.go

```diff
@@ -1246,7 +1246,7 @@ func (a *Operator) allNamespaceOperatorGroups() ([]*v1.OperatorGroup, error) {
 }
 
 func (a *Operator) syncOLMConfig(obj interface{}) (syncError error) {
-	a.logger.Debug("Processing olmConfig")
+	a.logger.Debugf("Processing olmConfig")
```

Comment:
I'm not sure why, but this log comes in at the info level on an openshift cluster, even though in the code it's at a debug level. Maybe this change would cause it to remain at the debug level and not appear in the logs, not sure.

Reply:
Yes, but looking at the logs in the olm operator, the logging level is set to info, so it seems unexpected
```
time="2022-02-15T16:17:33Z" level=info msg="log level info"
```

Reply:
Some possibilities:
- the user has set the debug flag to true on the olm-operator deployment in OpenShift (if the logs you're citing come from there, this would be unlikely)
- there is a version of OLM we released that has it set true by default (unlikely for the same reason)
- the logger instantiated here isn't the same one OLM's controller ultimately uses (we fail to pass the logger in somewhere)